src/ADS1115_WE.h

```diff
@@ -102,28 +102,113 @@ typedef enum ADS1115_STATUS_OR_START{
+     * deactivated with next value within limits.
+     *
+     * ADS1115_LATCH_DISABLED (default)
+     * ADS1115_LATCH_ENABLED
+    */
 	void setAlertLatch(ADS1115_LATCH latch);
+
+	/* Sets the alert pin polarity if active:
+     *
+     * Enable or disable latch. If latch is enabled the alarm pin will be active until the
+     * conversion register is read (getResult functions). If disabled the alarm pin will be
+     * deactivated with next value within limits.
+     *
+     * ADS1115_ACT_LOW  ->  active low (default)
+     * ADS1115_ACT_HIGH ->  active high
+    */
 	void setAlertPol(ADS1115_ALERT_POL polarity);
+
+	/* Choose maximum limit or maxium and minimum alert limit (window)in Volt - alert pin will
+     * be active when measured values are beyond the maximum limit or outside the window
+     * Upper limit first: setAlertLimit_V(MODE, maximum, minimum)
+     * In max limit mode the minimum value is the limit where the alert pin will be deactivated (if
+     * not latched)
+     *
+     * ADS1115_MAX_LIMIT
+     * ADS1115_WINDOW
+    */
 	void setAlertModeAndLimit_V(ADS1115_COMP_MODE mode, float hithres, float lothres);
+
+	/* Set the conversion rate in SPS (samples per second)
+     * Options should be self-explaining:
+     *
+     * ADS1115_8_SPS
+     * ADS1115_16_SPS
+     * ADS1115_32_SPS
+     * ADS1115_64_SPS
+     * ADS1115_128_SPS (default)
+     * ADS1115_250_SPS
+     * ADS1115_475_SPS
+     * ADS1115_860_SPS
+    */
 	void setConvRate(ADS1115_CONV_RATE rate);
+
+	/* Set continous or single shot mode:
+     *
+     * ADS1115_CONTINOUS  ->  continous mode
+     * ADS1115_SINGLE     ->  single shot mode (default)
+    */
 	void setMeasureMode(ADS1115_MEASURE_MODE mode);
+
+	/*
+	 * Set the voltage range of the ADC to adjust the gain
+	 * Please note that you must not apply more than VDD + 0.3V to the input pins!
+	 *
+	 * ADS1115_RANGE_6144  ->  +/- 6144 mV
+	 * ADS1115_RANGE_4096  ->  +/- 4096 mV
+	 * ADS1115_RANGE_2048  ->  +/- 2048 mV (default)
+	 * ADS1115_RANGE_1024  ->  +/- 1024 mV
+	 * ADS1115_RANGE_0512  ->  +/- 512 mV
+	 * ADS1115_RANGE_0256  ->  +/- 256 mV
+	*/
 	void setVoltageRange_mV(ADS1115_RANGE range);
+
+	/* Set the inputs to be compared
+     *
+     * ADS1115_COMP_0_1    ->  compares 0 with 1 (default)
+     * ADS1115_COMP_0_3    ->  compares 0 with 3
+     * ADS1115_COMP_1_3    ->  compares 1 with 3
+     * ADS1115_COMP_2_3    ->  compares 2 with 3
+     * ADS1115_COMP_0_GND  ->  compares 0 with GND
+     * ADS1115_COMP_1_GND  ->  compares 1 with GND
+     * ADS1115_COMP_2_GND  ->  compares 2 with GND
+     * ADS1115_COMP_3_GND  ->  compares 3 with GND
+    */
 	void setCompareChannels(ADS1115_MUX mux);
+
 	bool isBusy();
 	void startSingleMeasurement();
 	float getResult_V();
 	float getResult_mV();
+
+	/* With this function the alert pin will be active, when a conversion is ready.
+     * In order to deactivate, use the setAlertLimit_V function
```

Comment:
@wollewald this docstring mentions the `setAlertLimit_V` but i can't find it. what should the correct name be?

Reply:
This is simply a left over from first versions, which were not published. The setAlertLimit_V function does not exist (anymore). 
Internally the conversion ready alert is activated when the MSB of the upper alert limit is set to 1 (which means a negative value) and the MSB of the lower alert limit is set to "0" (which means a positive value). A bit strange, but this is how the ADS1115 works.
Therefore there are two ways to disable the conversion ready alert:
1) if you chose a combination of upper and lower limit which makes sense (upper limit higher than lower limit), using setAlertModeAndLimit_V --> then you use the alert pin for limit alert
2) setAlertPinMode(ADS1115_DISABLE_ALERT); --> then you don't use the alert pin at all
I am not sure if I should write this in detail into the comment. I think I will only mention option 2. What do you think?